data-otservbr-global/scripts/talkactions/player/server_info.lua

```diff
@@ -1,18 +1,43 @@
 local serverInfo = TalkAction("!serverinfo")
-
 function serverInfo.onSay(player, words, param)
-	local configRateSkill =  configManager.getNumber(configKeys.RATE_SKILL)
-	local baseRate = player:getFinalBaseRateExperience()
-	player:sendTextMessage(MESSAGE_EVENT_ADVANCE, "Server Info:"
-	.. "\nExp rate: " .. baseRate
-	.. "\nSword Skill rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_SWORD), configRateSkill)
-	.. "\nClub Skill rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_CLUB), configRateSkill)
-	.. "\nAxe Skill rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_AXE), configRateSkill)
-	.. "\nDistance Skill rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_DISTANCE), configRateSkill)
-	.. "\nShield Skill rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_SHIELD), configRateSkill)
-	.. "\nFist Skill rate: " .. getRateFromTable(skillsStages, player:getSkillLevel(SKILL_FIST), configRateSkill)
-	.. "\nMagic rate: " .. getRateFromTable(magicLevelStages, player:getBaseMagicLevel(), configManager.getNumber(configKeys.RATE_MAGIC))
-	.. "\nLoot rate: " .. configManager.getNumber(configKeys.RATE_LOOT))
+	local exprate = configManager.getNumber(configKeys.RATE_EXPERIENCE)
```

Comment:
why are creating a variable, if you don't use it?

Reply:
I don't understand why you're saying if I'm not going to use this code, I created it to improve optimization and internal organization. Every pull request I submit is for my own use and to function correctly and contribute to the improvement of the code.